src/main/java/net/dries007/tfc/objects/items/metal/ItemMetal.java

```diff
@@ -47,9 +48,9 @@ public ItemMetal(Metal metal, Metal.ItemType type)
         TABLE.get(metal).put(type, this);
 
         setNoRepair();
-        OreDictionaryHelper.register(this, type);
+        OreDictionaryHelper.registerMeta(this, OreDictionary.WILDCARD_VALUE, type);
```

Comment:
This is pointless. The two calls do the exact same thing if `OreDictionary.WILDCARD_VALUE` is passed in. Additionally, this makes it seem like `ItemMetal` uses metadata, which is kind of does, but technically really shouldn't be, and won't be in 1.13+